data/mayurstwt.json

```
@@ -0,0 +1,19 @@
+{
+  "name": "Mayur Sankhala",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "Happy since learned development",
+  "avatar": "https://github.com/mayurstwt.png"
+  "links": [
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/mayurstwt",
+      "icon": "FaTwitter"
+    },
+    {
+      "name": "Follow me on Github",
+      "url": "https://github.com/mayurstwt",
+      "icon": "FaTwitter"
```

Non-blocking: You might want to use `FaGithub` for this link, I guess.
```suggestion
      "icon": "FaGithub"
```